chef_master/source/vmware.rst

```diff
@@ -0,0 +1,527 @@
+=====================================================
+Chef and VMware
+=====================================================
+
+`[edit on GitHub] <https://github.com/chef/chef-web-docs/blob/master/chef_master/source/vmware.rst>`__
+
+.. note:: This page collects information about Chef that is specific to using Chef with VMware.
+
+VMware, Inc. is a subsidiary of Dell Technologies that provides cloud computing and platform
+virtualization software and services. It was the first commercially successful company to
+virtualize the x86 architecture.
+
+VMware's desktop software runs on Microsoft Windows, Linux, and macOS, while its enterprise software
+hypervisor for servers, VMware ESXi, is a bare-metal hypervisor that runs directly on server
+hardware without requiring an additional underlying operating system. [Wikipedia]_
+
+We also attempt to centralize the discussions for VMware and Chef on the `[VMware{code}] <https://code.vmware.com/web/code/join>`__ Slack team. We are located in **#chef**.
+
+knife
+=====================================================
+
+We have knife plugins with multiple applications in the VMware stack. These following are the
+ones that are directly supported by Chef.
+
+knife-vsphere
+-----------------------------------------------------
+
+`[Github] <https://github.com/chef-partners/knife-vsphere>`__
+
+* Supports vCenter > 5.0
+* Most VMware compute use cases are covered
+* This integration is the main starting point for Chef and VMware
+
+The main settings for your `knife.rb` are these:
+
+.. code-block:: ruby
+
+   knife[:vsphere_host] = "vcenter-hostname"
+   knife[:vsphere_user] = "privileged username" # Domain logins may need to be "user@domain.com"
+   knife[:vsphere_pass] = "your password"       # or %Q(mypasswordwithfunnycharacters)
+   knife[:vsphere_dc] = "your-datacenter"
+   knife[:vsphere_insecure] = true              # Set this if you have self signed certs
+
+With these settings this should be all you need to connect to your vSphere cluster.
+
+Here are some basic usage examples to help get you started.
+
+This clones from a VMware template and bootstraps chef into it. It uses the generic DHCP options.
+
+.. code-block:: bash
+
+   $ knife vsphere vm clone MACHINENAME --template TEMPLATENAME --bootstrap --cips dhcp
+
+This clones a vm from a VMware template bootstraps chef, then uses a Customization template
+called "SPEC" to help bootstrap. Also calls a different SSH user and Password.
```

Comment:
Nevermind, I was reading this wrong, thought it was referring to the example above it, instead of the one below.